api/src/test/java/org/pmiops/workbench/firecloud/FireCloudServiceImplTest.java

```diff
@@ -55,16 +57,31 @@
   private NihApi nihApi;
   @Mock
   private StatusApi statusApi;
+  @Mock
+  private GoogleCredential fireCloudCredential;
+  @Mock
+  private ServiceAccounts serviceAccounts;
+  @Mock
+  private GoogleCredential impersonatedCredential;
+
+  private WorkbenchConfig workbenchConfig;
 
   @Rule
   public MockitoRule mockitoRule = MockitoJUnit.rule();
 
   @Before
   public void setUp() {
+    workbenchConfig = new WorkbenchConfig();
```

Comment:
Could you use the new `createEmptyConfig` here and then override what you want to override?

Reply:
This is a good idea, but this change shouldn't have been in the PR to begin with... I'll consider following-up with a cleanup on all unit tests which had been manually constructing WorkbenchConfigs.